.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 5,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 6,
-            "hours": [
-              0,
-              3,
-              6,
-              9,
-              12,
-              15,
-              18,
-              21
-            ],
-            "timezoneOffset": -8
-          }
-        ],
-        "searchTerms": [
-          {
-            "name": "isIssue",
-            "parameters": {}
-          },
-          {
-            "name": "isOpen",
-            "parameters": {}
-          },
-          {
-            "name": "hasLabel",
-            "parameters": {
-              "label": "duplicate :busts_in_silhouette:"
-            }
-          },
-          {
-            "name": "noActivitySince",
-            "parameters": {
-              "days": 1
-            }
-          }
-        ],
-        "actions": [
-          {
-            "name": "addReply",
-            "parameters": {
-              "comment": "This issue has been marked as duplicate and has not had any activity for **1 day**. It will be closed for housekeeping purposes."
-            }
-          },
-          {
-            "name": "closeIssue",
-            "parameters": {}
-          }
-        ]
-      },
-      "id": "p6t3pVDqNP"
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "RKZiW8qGn",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isAction",
-              "parameters": {
-                "action": "opened"
-              }
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isPartOfProject",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "operator": "not",
-              "operands": [
-                {
-                  "name": "isAssignedToSomeone",
-                  "parameters": {}
-                }
-              ]
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "titlePattern": "Feature",
-                "label": "feature request :mailbox_with_mail:"
-              }
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Add Voting to new Feature Requests",
```

Comment:
This feature didn't get ported over because `addReactionToIssue` is not supported.